pkgs/applications/video/mpv/scripts/sponsorblock.nix

```diff
@@ -0,0 +1,49 @@
+{ stdenv, fetchFromGitHub, python3 }:
+
+stdenv.mkDerivation rec {
+  pname = "mpv_sponsorblock";
+  version = "f71e49e"; # 2020-07-05
+
+  src = fetchFromGitHub {
+    owner = "po5";
+    repo = "mpv_sponsorblock";
+    rev = version;
+    sha256 = "1fr4cagzs26ygxyk8dxqvjw4n85fzv6is6cb1jhr2qnsjg6pa0p8";
+  };
+
+  dontBuild = true;
+
+  postPatch = let
+    # This plugin needs to write the assigned user ID and its blocklist somewhere.
+    # By default, it attempts to write those files at the location of the scripts,
+    # which isn't possible because the nix store is read-only.
+    dataDir = "/tmp";
```

Comment:
This location might be problematic on a multi-user system.
Using the user's home directory doesn't work because `~/` doesn't seem to be recognised.
It would also be nice to put those data under `~/.cache/mpv/scripts/sponsorblock/` but the script doesn't seem to create intermediate directories by itself.

Reply:
Maybe something worth mentioning and fixing upstream?

Reply:
I think it would be adequate to make a PR upstream to have the lua code lookup `$XDG_DATA_DIR/mpv/scripts/sponsorblock.txt` or default to the old behaviour. Then we can use such PR as patch waiting à release.

Reply:
PR opened (https://github.com/po5/mpv_sponsorblock/pull/17) and patches added to the package.